### DISCUSSION TITLE
Setting image tag alt attribute values

### DISCUSSION THREAD

**Discussion by chrisdenman** [learnyouahaskell/learnyouahaskell.github.io#78] [General]
Hi,

As part of functionality testing the site, I've noticed that we don't really set image alt attribute values with any meaningful values. They are currently set to 'nonsense' but funny values at best.

Most of the images don't really add to the content but if someone is using a screen reader to access the site, the image alternative text values will at best annoy them and may confuse them.

Do you think there's value in providing suitable image alt attribute values to help people that have differing accessibility requirements?

I understand it is a long shot that people with visual impairments will be using the site with a screen reader but having decent values for these fields may help with content-based searching and provide a nudge in the right direction.

I'm happy to have a go at it. Any input would be appreciated.

Thanks!

**Comment by pierluc-codes** [comment]
Hey @chrisdenman,

Thanks for the feedback (and sorry about the delay!). I think it would be great addition to the document.  [Markdown is supporting alt](https://www.markdownguide.org/cheat-sheet/) :)

Feel free to submit a contribution.

**Reply by chrisdenman** [reply]
Thanks, i'll convert it into an issue to keep it remembered.
